scripts/lint_python_docstrings.sh

```diff
@@ -4,21 +4,28 @@ set -euo pipefail
 # Lint docstrings only against the the diff to avoid too many errors.
 # Check only production code. Ignore other other errors which are captured by `lint`
 
+echo "1-here"
 # Compare against `main` if no branch was provided
 BRANCH="${1:-main}"
+
+echo "2-here"
 # Diff of committed changes (shows only changes introduced by your branch
 FILES_WITH_DIFF=`git diff $BRANCH...HEAD --name-only -- rasa | grep -e '\.py$' | xargs echo -n`
```

Comment:
this seems to fail

Reply:
that's the offending commit https://github.com/RasaHQ/rasa/commit/4aefb5fce536a2e33a9573c887c00d84a8c93012